package.json

```diff
@@ -6,9 +6,18 @@
   "scripts": {
     "build": "NODE_ENV=development node_modules/webpack/bin/webpack.js --progress",
     "build:production": "NODE_ENV=production node_modules/webpack/bin/webpack.js --progress",
-    "lint": "node_modules/eslint/bin/eslint.js src/ --ext js --ext json",
+    "clean": "rm -rf .jest && rm -rf node_modules",
+    "cover": "node_modules/jest/bin/jest.js --coverage",
     "format": "node_modules/eslint/bin/eslint.js src/ --fix --ext js --ext json",
-    "test": "mocha src/test/*/*.js --compilers js:babel-register"
+    "lint": "node_modules/eslint/bin/eslint.js src/ --ext js --ext json",
+    "test": "node_modules/jest/bin/jest.js"
```

Comment:
why not just 

```sh
"test": "jest"
```

The same for eslint. Afaik, npm does the look up in local node_modules automatically.